en/docs/develop/components/api-proxies/api-proxy.md

```diff
@@ -41,7 +41,7 @@ Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to its
 
 Once you deploy the REST API to the development environment, it is then exposed by an API Gateway with API security turned on. Developers get a testable URL for their API which they can use to verify the functionality of the API they implemented.
 
-You may set up a separate environment to test the API proxy before pushing it to production. To deploy the API proxy to both the test environment and the development environment, enter the test environment URL as the sandbox endpoint and the API backend URL as the endpoint.
+You may set up a separate sandbox environment to test the API proxy before pushing it to production. To deploy the API proxy to both the sandbox environment and the development environment, enter the test environment URL as the sandbox endpoint and the API backend URL as the endpoint.
```

Comment:
enter the test environment -> enter the sandbox environment

Reply:
Can we incorporate these as well https://github.com/wso2-enterprise/choreo/issues/17227#issuecomment-1378375756?

Just to add a header and mention the 2 ways it can be configured?

Reply:
Add a  screenshot showing the configurables side panel which opens when you click deploy/promote or edit configurables in env card

Reply:
+1 for enter the test environment -> enter the sandbox endpoint